README.md

```diff
@@ -81,7 +81,12 @@ See [GiftedMessengerExample/GiftedMessengerContainer.js](https://raw.githubuserc
 | senderName                    | String   | Name of the sender of the messages                                         | Both     | 'Sender'                         |
 | styles                        | Function | Styles of children components - See GiftedMessenger.js/componentWillMount  | Both     | {}                               |
 | submitOnReturn                | Boolean  | Send message when clicking on submit                                       | Both     | false                            |
-| typingMessage                 | String   | Display a text at the bottom of the list. Eg: 'User is typing a message'   | Both     | ''                               |
+| typingMessage                 | String   | Display a text at the bottom of the list. Eg: 'User is typing a message'   | Both     | ''    
+| textInputProps                | Object   | Props passed to Textinput                                                  | Both     | {}  
+| multiline                     | Boolean  | When true, textinput expands vertically on return                          | Both     | false  
```

Comment:
Should `multiline` be here? TextInput props include `multiline`, maybe to avoid confusion make a note here. Not too important.